sample/cf-nodejs-logging-support-sample/app.js

```diff
@@ -89,9 +102,23 @@ app.get("/correlationandtenantid", function (req, res) {
   var correlationId = reqLogger.getCorrelationId();
   var tenantId = reqLogger.getTenantId();
   reqLogger.info("Correlation ID: %s Tenant ID: %s", correlationId, tenantId);
-  res.send();
+  res.send(lastMessage);
 });
 
+app.get("/binding_information", function (_, res) {
+  result = "There are the following bindings: <br>"
+  if (process.env.VCAP_SERVICES)
+    var services = JSON.parse(process.env.VCAP_SERVICES)
```

Comment:
I would not define a variable inside a block and then consume it outside. Technically, this is working for a `var` variable as it is function scoped but it looks a bit strange.

Reply:
definition is factored out